documentation/book/assembly-kafka-rack.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-rack-{context}']
+
+= Kafka rack awareness
+
+{ProductName} allows you to enable Kafka rack awareness.
+The rack awareness feature helps to spread the Kafka broker pods and Kafka topic replicas across different racks.
+Enabling rack awareness helps to improve availability of Kafka brokers and the topics they are hosting.
+Rack might represent an availability zone, data center or actual rack in your data center.
```

Comment:
We might want to make this a note and put Rack in quotes since we're talking about the term "Rack"